pkgs/tools/misc/zsjed/default.nix

```diff
@@ -0,0 +1,68 @@
+{ coreutils
+, diffutils
+, fetchurl
+, findutils
+, gawk
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zsjed";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/zsjed-${version}.tar.gz";
+    sha256 = "1rcrvrqjsv80ldz7cscqbrf98jb0yd2in97irrzlpask699z5k29";
+  };
+
+  nativeBuildInputs = [
+    diffutils
+  ];
+
+  buildInputs = [
+    coreutils # sort
+    findutils
+    gawk
+  ];
+
+  doCheck = true;
+  enableParallelBuilding = true;
+
+  # doesn't test zsjed-mkls
+  checkPhase = ''
+    mkdir tmp
+    cd tmp
+    touch f1 f2
+    echo f1 > l
+    echo f2 >> l
+    ../zsjed-clear l
+    echo "a b c d e f g h i j k" > i
+    ../zsjed-spread i l
+    ../zsjed-collect j l
+    cmp i j
+    cd ..
+    rm -rf tmp
```

Comment:
```suggestion
```

Reply:
Huh, does Nix automatically remove any temp folder created by `mktemp -d`?

Reply:
nix-collect-garbage deletes it.